shotgun_api3/shotgun.py

```diff
@@ -2895,6 +2895,43 @@ def nav_expand(self, path, seed_entity_field=None, entity_fields=None):
             }
         )
 
+    def nav_search_string(self, path_filter, search_string):
```

Comment:
this is probably my fault because i was using `path_filter` when we were chatting - sorry! Seeing it alongside `def nav_expand(self, path, seed_entity_field=None, entity_fields=None)` i now wonder if it's better to call it 
`def nav_search_string(self, path, search_string):` for consistency. What do you think? `root_path` is another suggestion to emphasize that we are requesting a search  below the specified path tree